Processing/Explore_New_Parks.Rmd

```diff
@@ -0,0 +1,303 @@
+---
+title: "Data_Availability_Parks"
+author: "KEC"
+date: "2024-11-05"
+output: word_document
+editor_options: 
+  chunk_output_type: inline
+---
+
+Define park units that you want to explore data for
+
+```{r}
+
+parks <- c("VALL","CHIR", "FOBO", "CORO", "CACH", "HUTR", "NAVA", "ZION", "CEBR")
+```
+
+
+```{r setup, include=FALSE}
+source("setup.R")
+
+pal <- c("black",
+            "#dca237", 
+    "#6eb2e4", 
+   "#459a74", 
+  "#eee461", 
+   "#3070ad", 
+  "#c66426", 
+   "#c07da5",
+  "#ECD8E4"
+)
+
+```
+
+# Input water supply table points
+
+```{r}
+
+table_path <- "data/Water_Supply_Systems/NPS_Water_Systems_Database.xlsx"
+
+
+
+# Supply table (Currently filtered to BRCA)
+supply_table <- read_excel(table_path, sheet = 2, na = "NA") %>%
+  janitor::clean_names() %>%
+  dplyr::filter(region %in% c("Pacific West Region", "Intermountain Region")) %>%
+  dplyr::filter(park_unit %in% parks)
+
+# Source table
+source_table <- read_excel(table_path, sheet = 3, na = "NA") %>%
+  janitor::clean_names() %>%
+  #dplyr::filter(state == "UT")
+  dplyr::filter(wsd_system_id %in% supply_table$wsd_system_id) %>%
+  dplyr::mutate(well_depth = as.numeric(well_depth))
+
+# Copy of source table as sf_object - some columns without location data are
+# dropped. Circle back to this later.
+source_table_locs <- source_table %>%
+  drop_na(c("source_longitude", "source_longitude")) %>%
+  st_as_sf(coords = (c("source_longitude","source_latitude")), 
+           crs = 4326,
+           remove = FALSE) 
+
+park_boundaries <- getParkSystem() %>%
+  dplyr::filter(UNIT_CODE %in% parks)
+```
+
+
+Plot all park boundaries and water supply points
+```{r}
+
+a <- mapview(park_boundaries,
+          col.regions = "seagreen",
+          alpha.regions = 0.3,
+          layer.name = "Park Boundaries") +
+  mapview(source_table_locs,
+        zcol = "source_type",
+        cex = 4,
+        layer.name = "Source Type") 
+
+  leafem::addStaticLabels(a,
```

Comment:
This is fun and I've never used before! I think this package will need to get added to setup.R script?

Reply:
Interesting that a leaflet package works on mapview objects lol